packages/jest-worker/src/types.ts

```diff
@@ -5,6 +5,9 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import EventEmitter from 'events';
```

Comment:
Should this be `import * as EventEmitter from 'events'` ?

Reply:
It's the same object

```sh-session
$ node -p "require('events') === require('events').EventEmitter"
true
```

And the node docs does not destructure: https://nodejs.org/api/events.html#events_events

Reply:
My concern is that `events` may not has a default export like `path`.
import path from 'path` won't work on ts.

Reply:
Fair point! Changed. Should we add `"allowSyntheticDefaultImports": false` to out tsconfig?

Reply:
That would conflict with `esModuleInterop: true`. I think we need it for now, but in an ideal world in the future where everything is written in ESM we will be able to drop `esModuleInterop`

Reply:
I was allowed to add it below `esModuleInterop` (usually tsc will complain about conflicting options). I didn't fix/ignore the errors and try to run anything if it changed the semantics

Reply:
Anyways, this PR is not the right place to discuss it. Might be worth a new issue, though, so we do it correctly

Reply:
`esModuleInterop` doc: "Emit __importStar and __importDefault helpers for runtime babel ecosystem compatibility and enable --allowSyntheticDefaultImports for typesystem compatibility"

Reply:
Yeah, but that doesn't mean I can't disable it again, does it?

Reply:
Might be possible to, but it seems like that would mean to forbid default-importing `module.exports` for type checking, but including the interop helper in output code. Because we use Babel, so our tsc does not emit anything, that would equal not enabling either of them at all, I think. And I believe we need it at least for default-importing some util libraries from npm that use `module.exports =`

Reply:
>Anyways, this PR is not the right place to discuss it. Might be worth a new issue, though, so we do it correctly

😀